internal/cmd/organization/member/member.go

```diff
@@ -14,8 +14,8 @@ import (
 func NewCmd(p *print.Printer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "member",
-		Short: "Provides functionality regarding organization members",
-		Long:  "Provides functionality regarding organization members.",
+		Short: "Provides functionality for organization members",
+		Long:  "Provides functionality for organization members.",
```

Comment:
This might be a bit misleading, do you think it would make sense something like `Manage members of an organization`?

Reply:
I agree

Reply:
Or `Manage organization members`